src/Components/teacherDashboard/applications.jsx

```diff
@@ -0,0 +1,113 @@
+import * as React from 'react';
+import { DataGrid } from '@material-ui/data-grid';
+import { projects } from '../../Data/projects';
+
+const columns = [
+  { field: 'id', headerName: 'ID', width: 90 },
+  {
+    field: 'firstName',
+    headerName: 'First name',
+    width: 150,
+    editable: true,
+  },
+  {
+    field: 'lastName',
+    headerName: 'Last name',
+    width: 150,
+    editable: true,
+  },
+  {
+    field: 'age',
+    headerName: 'Age',
+    type: 'number',
+    width: 110,
+    editable: true,
+  },
+  {
+    field: 'fullName',
+    headerName: 'Full name',
+    description: 'This column has a value getter and is not sortable.',
+    sortable: false,
+    width: 160,
+    valueGetter: (params) =>
+      `${params.getValue(params.id, 'firstName') || ''} ${
+        params.getValue(params.id, 'lastName') || ''
+      }`,
+  },
+  {
+    field: 'status',
+    headerName: 'Status',
+
+    width: 110,
+    editable: true,
+  },
+];
+
+const rows = [
+  { id: 1, lastName: 'Snow', firstName: 'Jon', age: 35, status: 'accepted' },
+  {
+    id: 2,
+    lastName: 'Lannister',
+    firstName: 'Cersei',
+    age: 42,
+    status: 'accepted',
+  },
+  {
+    id: 3,
+    lastName: 'Lannister',
+    firstName: 'Jaime',
+    age: 45,
+    status: 'accepted',
+  },
+  { id: 4, lastName: 'Stark', firstName: 'Arya', age: 16, status: 'accepted' },
+  {
+    id: 5,
+    lastName: 'Targaryen',
+    firstName: 'Daenerys',
+    age: null,
+    status: 'accepted',
+  },
+  {
+    id: 6,
+    lastName: 'Melisandre',
+    firstName: null,
+    age: 150,
+    status: 'accepted',
+  },
+  {
+    id: 7,
+    lastName: 'Clifford',
+    firstName: 'Ferrara',
+    age: 44,
+    status: 'accepted',
+  },
+  {
+    id: 8,
+    lastName: 'Frances',
+    firstName: 'Rossini',
+    age: 36,
+    status: 'accepted',
+  },
+  {
+    id: 9,
+    lastName: 'Roxie',
+    firstName: 'Harvey',
+    age: 65,
+    status: 'accepted',
+  },
+];
```

Comment:
Use the Data folder to store temporay data. Make a new json file there and import data from there.